ehrql/codes.py

```diff
@@ -30,6 +30,26 @@ def _to_primitive_type(self):
         return self.value
 
 
+# A base class for fields that are concatenated lists of clinical codes. This occurs
+# in the admitted patient care spell (apcs) table of hospital episode statistics for
+# all_diagnoses (ICD10 codes), and all_procedures (OPCS4 codes).
+#
+# This inherits from str because that's what the underlying data is, but is in this
+# file as it's sort of a code. In future a better implementation might be to parse the
+# field value into a Set of clinical codes.
+@dataclasses.dataclass(frozen=True)
```

Comment:
I don't think this actually needs to be a dataclass because it's not using any of the dataclass behaviour.

Reply:
I've removed it, and the tests still pass, so I'm assuming you're right. I'll need to read up on dataclasses to understand it better.